tests/services/inventory/test_data/additional_cai_iam_policies.dump

```diff
@@ -3,7 +3,7 @@
 # update_cai_dumps.py script. Any lines that start with a '#' are ignored.
 #
 # Bigquery Dataset with project number
-{"name":"//bigquery.googleapis.com/projects/1042/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset","iam_policy":{"bindings":[{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project2"]},{"role":"roles/bigquery.dataOwner","members":["projectOwner:project2","user:user@forseti.testing"]},{"role":"roles/bigquery.dataViewer","members":["allAuthenticatedUsers","domain:forseti.testing","projectViewer:project2"]}]}}
+{"name":"//bigquery.googleapis.com/projects/1042/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset","iam_policy":{"bindings":[{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project2"]},{"role":"roles/bigquery.dataOwner","members":["projectOwner:project2","user:user@forseti.testing"]},{"role":"roles/bigquery.dataViewer","members":["allAuthenticatedUsers","domain:forseti.testing","projectViewer:project2"]},{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project_does_not_exist"]}]}}
```

Comment:
The dataEditor role already exists, you can just add this new member to the members list.

Reply:
Done, PTAL